js/north.js

```diff
@@ -44,79 +44,6 @@
         } );
     };
 
-    $.fn.burstMenuHover = function( options ) {
```

Comment:
How are you creating this menu hover functionality without this JS? Is it using CSS3 animations now? However you're doing it, it seems a lot more light weight.